specification/schemas.json

```diff
@@ -119,6 +125,12 @@
   "PickupDropoffLocation": {
     "$ref": "./schemas/PickupDropoffLocation.json"
   },
+  "Predicate": {
```

Comment:
Should be placed after `PhysicalProperties` to keep the list ordered alphabetically.